sympy/solvers/tests/test_solveset.py

```diff
@@ -2472,6 +2472,14 @@ def test_issue_17276():
      FiniteSet((5**(S(1)/5), 25*5**(S(3)/10)))
 
 
+def test_issue_10426():
+    x=Dummy('x')
+    a=Symbol('a')
+    n=Symbol('n')
+    assert solveset(sin(x + a) - sin(x), a) == \
+    Union(ImageSet(Lambda(n, 2*pi*n), S.Integers),Intersection(S.Complexes,ImageSet(Lambda(n, -I*(I*(2*n*pi + arg(-exp(-2*I*x))) + 2*im(x))), S.Integers)))
```

Comment:
Would be more readable as
```suggestion
    x = Dummy('x')
    a = Symbol('a')
    n = Symbol('n')
    assert solveset(sin(x + a) - sin(x), a) == Union(
        ImageSet(Lambda(n, 2*pi*n), S.Integers),
        Intersection(S.Complexes, ImageSet(Lambda(n, -I*(I*(2*n*pi + arg(-exp(-2*I*x))) + 2*im(x))), S.Integers)))
```

Reply:
Thank you for the suggestion, I tried it running locally, it shows an assertion error. I guess it is coming because of the order of n and pi in both are different. can you provide me a confirmation on that.